MovieApp/MovieApp/Models/Movie.swift

```diff
@@ -0,0 +1,7 @@
+struct Movie: Decodable {
+    
+    let title: String
+    let desription: String
+    let imageString: String
```

Comment:
Please rename this to reflect the data it will hold.
`imageString` could be the name of the image or the binary data written in string.
Some suggestions:
```swift
imageName
imageUrl
imagePath
```